Marlin/src/pins/pins_GT2560_REV_A_PLUS.h

```diff
@@ -29,4 +29,8 @@
 #undef BOARD_NAME
 #define BOARD_NAME  "GT2560 Rev.A+"
 
+#if ENABLED(BLTOUCH)
+#define SERVO0_PIN  32
+#elif
```

Comment:
`#elif` is a typo.

Reply:
I found the elif in marlin code, doesn't it work in the manner I'm using it?